packages/form/src/components/DateField/index.tsx

```diff
@@ -101,11 +101,11 @@ export const DateField = ({
           input.onChange(newDate)
         }
       }}
-      onBlur={(e: FocusEvent<HTMLElement, Element>) => {
+      onBlur={(e: FocusEvent<HTMLElement>) => {
```

Comment:
I believe we shouldn't have to type again the parameters

Reply:
@QuiiBz 😢 

<img width="1124" alt="Screenshot 2023-03-02 at 15 49 45" src="https://user-images.githubusercontent.com/2802867/222462480-3e974fb1-143f-42ba-ba8a-5ae2842882b1.png">